### PR TITLE
Add info on practices and prereqs keys

### PR DESCRIPTION
**Pull request by iHiD**
We should also link back here from https://github.com/exercism/docs/blob/main/anatomy/tracks/config-json.md#practice-exercises. I think this belongs better in this document where someone is in the mindset of thinking about Practice Exercises.

### CODE REVIEW

File: anatomy/tracks/practice-exercises.md [hunk 1]
```diff
@@ -25,6 +25,23 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
 }
 ```
 
+### `practices`
+
+The `practices` key should list the slugs of Concepts that this Practice Exericse actively allows a student to practice.
```
Comment (ErikSchierboom) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
The `practices` key should list the slugs of Concepts that this Practice Exercise actively allows a student to practice. 
```

File: anatomy/tracks/practice-exercises.md [hunk 2]
```diff
@@ -25,6 +25,23 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
 }
 ```
 
+### `practices`
+
+The `practices` key should list the slugs of Concepts that this Practice Exericse actively allows a student to practice. 
+
+- These show up in the UI as "Practice this Concept in: TwoFer, Leap, etc`
```
Comment (ErikSchierboom) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
- These show up in the UI as "Practice this Concept in: TwoFer, Leap, etc"
```

File: anatomy/tracks/practice-exercises.md [hunk 3]
```diff
@@ -25,6 +25,23 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
...
+- Try and choose 3 - 8 Exercises for this key.
```
Comment (ErikSchierboom) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
- Try and choose 3 - 8 Exercises per Concept.
```

Comment (iHiD) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
- Try and choose 3 - 8 Exercises that practice each Concept.
```

File: anatomy/tracks/practice-exercises.md (hunk 1 above)
Comment (cmccandless) [inline anatomy/tracks/practice-exercises.md:30]:
```suggestion
The `practices` key should list the slugs of Concepts that this Practice Exercise actively allows a student to practice. 
```

File: anatomy/tracks/practice-exercises.md [hunk 4]
```diff
@@ -25,6 +25,23 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
...
+- Try and choose at least two Exercises that allow someone to practice the basics of a Concept.
+- Some Concepts are very common (for example `strings` or `integers`). In those cases we recommend choosing a few good exercises that make people think about those Concepts in interesting ways. For example, exercises that require utf8, string concatenation, char enumeration, base-changes, etc, would all be good examples.
```
Comment (ErikSchierboom) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
- Some Concepts are very common (for example `strings`). In those cases we recommend choosing a few good exercises that make people think about those Concepts in interesting ways. For example, exercises that require UTF-8, string concatenation, char enumeration, etc, would all be good examples.
```

**Review by ErikSchierboom — APPROVED** [review]

File: anatomy/tracks/practice-exercises.md [hunk 5]
```diff
@@ -25,6 +25,23 @@ Practice Exercise metadata is defined in the `exercises.practice` key in the [co
...
+- Try and choose at least two Exercises that allow someone to practice the basics of a Concept.
+- Some Concepts are very common (for example `strings`). In those cases we recommend choosing a few good exercises that make people think about those Concepts in interesting ways. For example, exercises that require UTF-8, string concatenation, char enumeration, etc, would all be good examples.
+
+### `prerequisites`
+
+The `prerequisites` key lists the Concept Exercises that a student must have completed in order to access this Practice Exercise. 
+
+- These show up in the UI as "Learn Strings to unlock TwoFer"
+- It should include all Concepts that a student needs to have covered to be able to complete the exercise in at least one idiomatic way. For example, for the TwoFer exercise in Ruby, prerequisites might include `strings`, `optional-params`, `implicit-return`. 
+- For Exercises that can be completed using alternative Concepts (e.g. an Exercise solvable by `loops` or `recursion`), the maintainer should choose the approach that they would like to unlock the Exercise, considering the student's journey through the track.
```
Comment (iHiD) [inline anatomy/tracks/practice-exercises.md]:
```suggestion
- For Exercises that can be completed using alternative Concepts (e.g. an Exercise solvable by `loops` or `recursion`), the maintainer should choose the one approach that they would like to unlock the Exercise, considering the student's journey through the track. For example, the loops/recursion example, they might think this exercise is a good early practice of `loops` or that they might like to leave it later to teach recursion. They can also make use of an analyzer to prompt the student to try an alternative approach: "Nice work on solving this via loops. You might also like to try solving this using Recursion."
```

@angelikatyborska Is this clearer?

Comment (angelikatyborska) [reply anatomy/tracks/practice-exercises.md]: Yes

**Pull request merged**